src/main/java/frc/robot/ClawControl.java

```diff
@@ -38,13 +38,13 @@
     public static final double I = .002;
     public static final double REVERSEPOWER = .2;
     public static final double TIMERDELAY = 1;
+    DoubleSolenoid piston = new DoubleSolenoid(1, 2);
```

Comment:
Do you need this? Seems like the double solenoids for left and right are defined in the constructor below on line 43.